sources/pipedrive/__init__.py

```diff
@@ -184,16 +184,13 @@ def leads(
     )
     # Load leads pages sorted from newest to oldest and stop loading when
     # last incremental value is reached
-    last_value = update_time.last_value
     pages = get_pages(
         "leads",
         pipedrive_api_key,
         extra_params={"sort": "update_time DESC"},
     )
     for page in pages:
-        if last_value:
-            # Just check whether first item is lower, worst case we load 1 redundant page before break
-            first_item = page[0] if page else None
-            if first_item and first_item["update_time"] < last_value:
-                return
         yield rename_fields(page, fields_mapping)
+        # flag is set when first lower than start time is hits, no need to request more data
+        if update_time.start_out_of_range:
```

Comment:
to my knowledge this pipeline is already in production. we should keep the old version or detect and run the supported method

Reply:
Hmm ok best to leave the old version for now then. These changes in zendesk are incompatible also, should we be doing checks there?